Makefile

```diff
@@ -363,10 +363,60 @@ proto-gen:
 	@echo "Generating Protobuf files"
 	@$(protoImage) sh ./scripts/protocgen.sh
 
+SWAGGER_PROTO_DIR=./swagger-proto
+SWAGGER_THIRD_PARTY_DIR=$(SWAGGER_PROTO_DIR)/third_party
+SWAGGER_COSMOS_SDK_BRANCH=release/v0.47.x
+SWAGGER_IBC_BRANCH=release/v7.4.x
+proto-download-deps:
```

Comment:
To include cosmos-sdk and other third party queries and services into swagger, it requires download the raw proto files for generating proper swagger files.